src/index.html

```diff
@@ -4,9 +4,280 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        target="_blank"
+        class="logo-link"
+        >
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="index.html" class="is-active nav__link">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="card-container">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        >
+        <p class="card__title">APPLE A1419 iMac 27" Retina <br>
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product">Product code: 195434</p>
+        <div class="stars">
+          <div class="stars__stars stars__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="stars__text">
+            <p class="stars__card-text">Reviews: 5</p>
+          </div>
+        </div>
+        <div class="card__price">
+          <p class="card__text card__text--one">Price:</p>
+          <p class="card__text card__text--two">$2,199</p>
+        </div>
+        <button class="card__button" data-qa="card-hover">buy</button>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```